pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -636,6 +938,10 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 			// If the container failed the startup probe, we should kill it.
 			message = fmt.Sprintf("Container %s failed startup probe", container.Name)
 			reason = reasonStartupProbe
+		} else if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) &&
+			!m.computePodResizeAction(pod, idx, containerStatus, &changes) {
```

Comment:
note to myself: this is the key feature gate block to ensure that no actions occur during key loops when function is disabled.